src/main/kotlin/org/rust/ide/intentions/ConvertUFCSToMethodCallIntention.kt

```diff
@@ -0,0 +1,66 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.intentions
+
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiElement
+import org.rust.lang.core.psi.*
+import org.rust.lang.core.psi.ext.*
+
+class ConvertUFCSToMethodCallIntention : RsElementBaseIntentionAction<ConvertUFCSToMethodCallIntention.Context>() {
+    override fun getText() = "Convert to method call"
+    override fun getFamilyName() = text
+
+    data class Context(val callExpr: RsCallExpr, val function: RsFunction)
+
+    override fun findApplicableContext(project: Project, editor: Editor, element: PsiElement): Context? {
+        val call = element.ancestorStrict<RsCallExpr>(RsValueArgumentList::class.java) ?: return null
+        val path = (call.expr as? RsPathExpr)?.path ?: return null
+        val function = path.reference?.resolve() as? RsFunction ?: return null
+        if (!function.hasSelfParameters) return null
```

Comment:
```suggestion
        if (!function.isMethod) return null
```

`isMethod` also check that a function is an `impl`/`trait` member